inscontext/context.go

```diff
@@ -139,3 +140,14 @@ func WithLog(parent context.Context, clog core.Logger) *Ctx {
 func (ctx *Ctx) Log() core.Logger {
 	return ctx.log.WithField("traceid", ctx.TraceID())
 }
+
+// WithRandomTraceID returns *Ctx with random traceID (uuid format)
```

Comment:
randomID generation should live outside of context, in more clean implementation code would look like:

`inscontext.WithTraceID(ctx, cryptohelpers.RandTraceID()`